tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/compoundassignment/CompoundAssignmentTest.java

```diff
@@ -382,6 +382,7 @@ public void testCompoundAssignmentNegative() {
         BAssertUtil.validateError(compileResult, i++, "operator '>>>' not defined for 'int' and 'string'", 120, 5);
         BAssertUtil.validateError(compileResult, i++, "operator '+' not defined for 'int?' and 'int?'", 132, 5);
         BAssertUtil.validateError(compileResult, i++, "operator '+' not defined for 'int?' and 'int?'", 140, 5);
-        BAssertUtil.validateError(compileResult, i, "operator '+' not defined for 'int?' and 'int'", 150, 11);
+        BAssertUtil.validateError(compileResult, i++, "operator '+' not defined for 'int?' and 'int'", 150, 11);
+        BAssertUtil.validateError(compileResult, i, "invalid assignment in variable 'foo(bar)'", 156, 5);
```

Comment:
should we have a more specific error message? something like `function invocation are not allowed in lhs...`

Reply:
Fixed.